score-server/src/test/java/bio/overture/score/server/security/JWTSecurityTest.java

```diff
@@ -174,33 +204,33 @@ public void validateCanUploadWithValidAuthScopes() {
         return ResponseEntity.status(mvcResponse.getStatus()).body(responseObject);
     }
 
+    private final Map<ScopeOptions, List<String>> downloadScopesMap = Map.of(
+            VALID_SYSTEM,   List.of(resolveSystemDownloadScope(), "score.READ", "id.READ"),
```

Comment:
remove score.READ from this list.  Even if the system download scope will be `aws.READ`, its just misleading to have score.READ